README.md

```diff
@@ -39,35 +51,131 @@ class ListMovingVehicle(vqpy.QueryBase):
                                    select_cons=select_cons)
 ```
 
+We also have a utility function named `vqpy.utils.continuing`, which checks whether a condition function continues to be true for a certain duration. The following code demonstrate a query that selects all the `Person` objects whose `bottom_center` property lies in the specified region. Then the `track_id` field, `coordinate` field and `in_roi_periods` field. Note that `in_roi_periods` is generated by the `vqpy.utils.continuing` instance, meaning that the total time period the condition is satisfied.
+
+```python
+class People_loitering_query(vqpy.QueryBase):
+
+    @staticmethod
+    def setting() -> vqpy.VObjConstraint:
+        REGIONS = [[
+            (550, 550), (1162, 400), (1720, 720), (1430, 1072), (600, 1073)
+        ]]
+
+        filter_cons = {
+            "__class__": lambda x: x == Person,
+            "bottom_center": vqpy.utils.continuing(
+                condition=vqpy.utils.within_regions(REGIONS),
+                duration=10, name="in_roi"
+            ),
+        }
+        select_cons = {
+            "track_id": None,
+            "coordinate": lambda x: str(x),  # convert to string for JSON serialization
+            "in_roi_periods": None,
+        }
+        return vqpy.VObjConstraint(filter_cons, select_cons)
+```
+
 #### Launch the task
 
-Last, we can call `vqpy.launch` to start query video frames.
+After declaring `VObj` and `Query`, users should call the `vqpy.launch` function to deploy the video query, below is an example of this interface. Users should specify the detector class they expect, and map the object detection result to the defined VObj types.
 
 ```python
 vqpy.launch(cls_name=vqpy.COCO_CLASSES, # detection class
             cls_type={"car": Vehicle, "truck": Vehicle}, # mappings from detection class to VObj
             tasks=[ListMovingVehicle()], # a list of Queries to apply
             video_path=args.path, # the path of the queried video
-            save_folder=args.save_folder # result of query will be saved as a json file in this folder
+            save_folder=args.save_folder, # result of query will be saved as a json file in this folder
+            detector_name="yolox", # optional, specify the name of user's own model
+            detector_model_dir=args.pretrained_model_dir # optional, specify the directory of user's pretrained model
             )
 ```
 
-Under the hood, VQPy will automatically select an object detection model that outputs the specified `cls_name`. Multiple video optimizations will be conducted transparently to improve the end-to-end video query performance. 
+Under the hood, VQPy will automatically select an object detection model that outputs the specified `cls_name`, and decide the workflow to generate the object properties. Multiple video optimizations will be conducted transparently to improve the end-to-end video query performance.
 
-### Customization
+#### Detector and tracker
```

Comment:
Shall we include the detector and tracker in customization? Since users only need to know that when they need to customize and register their own detectors and trackers

Reply:
Should we put the sections of detector, tracker, and customization to the end of this README, or move them to a new document?

Reply:
I prefer a new document dedicated to customization. A shorter document might be easier for users to finish reading. And user can only read the customization document when they do need it.

Reply:
It is just my personal preference. Any ideas are welcome.

Reply:
I may add another document after adding `README.md`.